### DISCUSSION TITLE
How can `tailwindcss` be used to customize `radix-vue` themes?

### DISCUSSION THREAD

**Discussion by evanlong-me** [unovue/reka-ui#333] [General]
I might need a plan or a suggestion.

**Comment by evanlong-me** [comment]
I see [radix-ui](https://www.radix-ui.com/) has its own theme, but I think it is not very beautiful, I am more optimistic about the style of [shadcn-ui](https://ui.shadcn.com/) 🤣

**Comment by evanlong-me** [comment]
If there is a suitable plan, I think we can start another project to develop, Beautiful components based on `radix-vue`. 🤗

**Comment by zernonia** [comment]
`radix-vue` is design to provide primitive components only.. if you need a theme built on top of this.. Check out [Shadcn/Vue](https://www.shadcn-vue.com/)
